app/webpack/taxa/show/components/charts.jsx

```diff
@@ -253,61 +236,39 @@ class Charts extends React.Component {
   render( ) {
     const noHistoryData = _.isEmpty( this.props.historyKeys );
     const noSeasonalityData = _.isEmpty( this.props.seasonalityKeys );
-    const showLifeStage = _.find( this.props.seasonalityColumns, column => column[0].match( /Insect/ ) );
-    const insectLifeStageTab = (
-      <li role="presentation">
-        <a
-          href="#charts-insect-life-stage"
-          aria-controls="charts-insect-life-stage"
-          role="tab"
-          data-toggle="tab"
-          style={{ display: showLifeStage ? "block" : "none" }}
-        >
-          { I18n.t( "insect_life_stage" ) }
-        </a>
-      </li>
-    );
-    const insectLifeStagePanel = (
-      <div role="tabpanel" className="tab-pane" id="charts-insect-life-stage">
-        <div
-          className={
-            `no-content text-muted text-center ${noSeasonalityData ? "" : "hidden"}`
-          }
-        >
-          { I18n.t( "no_observations_yet" ) }
-        </div>
-        <div id="InsectLifeStageChart" className="SeasonalityChart FrequencyChart">
-        </div>
-      </div>
-    );
-    const showFlowering = _.find( this.props.seasonalityColumns, column => column[0].match( /Flowering/ ) );
-    const floweringPhenologyTab = (
-      <li role="presentation">
-        <a
-          href="#charts-flowering-phenology"
-          aria-controls="charts-flowering-phenology"
-          role="tab"
-          data-toggle="tab"
-          style={{ display: showFlowering ? "block" : "none" }}
-        >
-          { I18n.t( "flowering_phenology" ) }
-        </a>
-      </li>
-    );
-    const floweringPhenologyPanel = (
-      <div role="tabpanel" className="tab-pane" id="charts-flowering-phenology">
-        <div
-          className={
-            `no-content text-muted text-center ${noSeasonalityData ? "" : "hidden"}`
-          }
-        >
-          { I18n.t( "no_observations_yet" ) }
-        </div>
-        <div id="FloweringPhenologyChart" className="SeasonalityChart FrequencyChart">
-        </div>
-      </div>
-    );
-    // }
+    let fieldValueTabs = [];
+    let fieldValuePanels = [];
+    if ( this.props.chartedFieldValues ) {
+      _.each( this.props.chartedFieldValues, ( values, termID ) => {
+        fieldValueTabs.push( (
+          <li role="presentation" key={ `charts-field-values-${termID}` }>
+            <a
+              href={ `#charts-field-values-${termID}` }
+              aria-controls={ `charts-field-values-${termID}` }
+              role="tab"
+              data-toggle="tab"
+            >
+              { values[0].controlled_attribute.label }
```

Comment:
This should be translated if we have a translation for this term, so maybe `I18n.t( _.snakeCase( values[0].controlled_attribute.label ), { defaultValue: values[0].controlled_attribute.label } )`